src/Util/Annotation/Registry.php

```diff
@@ -62,7 +62,7 @@ public function forClassName(string $class): DocBlock
 
     /**
      * @throws Exception
-     * @psalm-param class-string $className
```

Comment:
Ow, not sure how this made it through 😱

Reply:
What's with this psalm-param? I do not really understand it, I copied it to more locations, seemed logical to me

Reply:
This is a restriction on the `string` type, stating that the string must be a class name. The `@psalm-param` is an annotation we have to use here, in lack of better type support in phpdocumentor/phpstorm/etc.

All good BTW 👍